web/generate-geometries.js

```diff
@@ -682,16 +681,25 @@ zoneDefinitions.forEach(zone => {
       'only once in zoneDefinitions');
   // Avoid zone with moreDetails
   if ( !('moreDetails' in zone) || !zone.moreDetails) {
-  	zones[zone.zoneName] = getDataForZone(zone, true);
+    zones[zone.zoneName] = getDataForZone(zone, true);
   }
 });
 
+// special case for GB, as we separate two islands from it
```

Comment:
@corradio it's the only zone for which we do manual manipulation. Do you think that code is OK? At least it works fine.

Reply:
let's move the code up to L236 then!